docs/product/profiling/getting-started.mdx

```diff
@@ -20,7 +20,7 @@ Profiling depends on Sentry's performance monitoring product being enabled befor
   - [iOS](/platforms/apple/guides/ios/profiling/)
   - [React Native](/platforms/react-native/profiling/) [beta]
   - [Flutter](/platforms/flutter/profiling/) [experimental, iOS and macOS only]
-  - [.NET](/platforms/dotnet/profiling/) [experimental, iOS only]
+  - [.NET](/platforms/dotnet/guides/maui/profiling/) [experimental, iOS only]
```

Comment:
Not sure why you want to steer users to the maui docs instead of the general dotnet docs? the pages look identical

Reply:
because these are the docs relevant to iOS profiling (what this line is about).

Reply:
ah gotcha